inputs/LiMetal_PorousSep_oxygen.yaml

```diff
@@ -121,43 +113,86 @@ phases:
   state:
     T: 300.0
     P: 1 atm
-- name: cathode
-  thermo: ideal-condensed
-  elements: [Li, C, O, Mn]
-  species: ['C[cathode]', 'Li2O2[cathode]', 'Cat[cathode]']
-  state:
-    X: {'C[cathode]': 0.80, 'Li2O2[cathode]': 0.00, 'Cat[cathode]': 0.20 }
-  standard-concentration-basis: unity
-  tabulated-species: Li2O2[cathode]
-- name: electrolyte
-  thermo: ideal-condensed
-  elements: [Li, P, F, C, H, O, E]
-  species: ['C3H4O3[elyt]', 'Li+[elyt]', 'PF6-[elyt]', O2(e), LiO2]
-  state:
-    X: {'C3H4O3[elyt]': 0.85464, 'Li+[elyt]': 0.07268,
-      'PF6-[elyt]': 0.07268}
-  standard-concentration-basis: unity
-  reactions: [electrolyte-reactions]
+
 - name: electron
   thermo: electron-cloud
   elements: [E]
   species: [electron]
   state:
     X: {electron: 1.0}
   density: 1.0 kg/m^3
+
 - name: lithium_electrolyte
   thermo: ideal-surface
   species: [(dummy)]
   kinetics: surface
   reactions: [lithium_electrolyte-reactions]
   site-density: 0.01 mol/cm^2
+  
+- name: electrolyte
+  thermo: ideal-condensed
+  elements: [Li, P, F, C, H, O, E]
+  species: ['C10H22O5[elyt]', 'Li+[elyt]', 'TFSI-[elyt]', O2(e), LiO2]
+  state:
+    X: {'C10H22O5[elyt]': 0.85464, 'TFSI-[elyt]': 0.07268,
+      'PF6-[elyt]': 0.07268, 'O2(e)': 0.0, 'LiO2': 0.0} 
```

Comment:
Probably easier, numerically, to have small amounts of O2(e) and LiO2, like 1e-6 at least.  Also, we need to have a consistent naming convention, here - some species have `[elyt]`, some have `(e)`, and some have nothing, to indicate the parent phase.  I like `(e)` the best, personally.

Reply:
just bc it takes less time to replace I went with [elyt]